src/pages/questionPage.js

```diff
@@ -65,3 +67,25 @@ const nextQuestion = () => {
   }, 1500);
   
 };
+
+// add pressure timer to quiz
+
+export const appendFullTimeCounter=()=>{
+  const fullTimeElement=createFullTime();
+  document.body.appendChild(fullTimeElement);
```

Comment:
can you append this into div that has 'user-progress' class and you can define this appending issue inside createFullTime method. And can you also add a class name for this element because we will be able to make css settings later.

Reply:
Mones can you change the name of method to 'createTimeElement'? you are going to be angry to me :-p